acoustid/future/fpindex/client.py

```
@@ -0,0 +1,396 @@
+            Fingerprint information
+        """
+        _, result = await self._request(
+            "GET",
+            f"/{index_name}/{fingerprint_id}",
+            expected_status=[200],
+            expected_response=GetFingerprintResponse,
+        )
+        return result
+
+    async def update_fingerprint(
+        self, index_name: str, fingerprint_id: int, hashes: list[int]
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)